packages/hooks/use-infinite-scroll/src/index.ts

```diff
@@ -161,6 +161,7 @@ export const useInfiniteScroll = <T extends HTMLElement = HTMLDivElement>({
   const isMountedRef = useRef<boolean>(false)
   const prevScrollPosition = useRef<number>(0)
   const [isFinish, setIsFinish] = useState<boolean>(false)
+  const [isLoad, setIsLoad] = useState<boolean>(false)
```

Comment:
Doesn't this manage the same status as `processingRef`?

Reply:
That’s correct. Thank you. I’ve made the corrections, so please review it again.🙇‍♂️

Reply:
I think you need to change it to `useState`.If it is `ref`, it will not be re-rendered when changed.

Reply:
Would it be better to change processingRef to useState?

Reply:
I think so. Otherwise `aria-busy` will not be updated.